src/lib/InputComponent.tsx

```diff
@@ -175,8 +175,19 @@ export class InputComponent extends React.Component<Props, State> {
     }
   };
 
-  handleChange = (event: NativeSyntheticEvent<TextInputChangeEventData>) => {
-    let value = event.nativeEvent.text;
+  handleChangeFromInput  = (event: NativeSyntheticEvent<TextInputChangeEventData>) => {
```

Comment:
Doesn't look like prettier ran on this like (there's a double space before the = sign)